python/pip_install/pip_repository.bzl

```diff
@@ -332,7 +336,10 @@ def _pip_repository_impl(rctx):
         for p in opt.split(" "):
             tokenized_options.append(p)
 
-    options = tokenized_options + rctx.attr.extra_pip_args
+    options = [
+        envsubst(pip_arg, rctx.attr.envsubst, rctx.os.environ)
```

Comment:
If the pip index url has the token in it, this will leak the value to `requirements.bzl` file in the hub repo and if it is vendored, it will leak it into the vendored file, just having the un-substituted value here and do envsubst in `_whl_library_impl` should be sufficient.

Consider using this feature in `pip_parse_vendored` example to ensure that this is not happening.

Reply:
Done.